ci/jenkins/test.sh

```diff
@@ -519,6 +524,23 @@ function deliver_antrea_windows {
     echo "==== Finish building and delivering Windows images ===="
 }
 
+function copy_kind_test_image {
+    common_images=("registry.k8s.io/e2e-test-images/agnhost:2.40" \
```

Comment:
Could you double-check if the agnhost version should be 2.45 in the current tests?

Reply:
Sure